browser/ui/webui/brave_rewards/rewards_page_handler.h

```diff
@@ -85,7 +86,7 @@ class RewardsPageHandler : public mojom::RewardsPageHandler {
   void GetAdsSettings(GetAdsSettingsCallback callback) override;
   void GetAdsStatement(GetAdsStatementCallback callback) override;
   void GetAdsHistory(GetAdsHistoryCallback callback) override;
-  void SetAdTypeEnabled(brave_ads::mojom::AdType ad_type,
+  void SetAdTypeEnabled(brave_ads::mojom::AdType mojom_ad_type,
```

Comment:
Let's leave this parameter name `ad_type`.

Reply:
Sure, changed back, thanks